main.go

```diff
@@ -446,24 +446,25 @@ func main() {
 		room := ctx.Param("room")
 		upgrader := websocket.Upgrader{
 			CheckOrigin: func(r *http.Request) bool {
-				// Allow specific origin
 				origin := r.Header.Get("Origin")
-				return strings.HasPrefix(origin, "http://localhost") || strings.HasPrefix(origin, "http://127.0.0.1") || strings.HasPrefix(origin, "file://")
+				return strings.HasPrefix(origin, "http://localhost") ||
+					strings.HasPrefix(origin, "http://127.0.0.1") ||
+					strings.HasPrefix(origin, "file://")
 			},
-
 			ReadBufferSize:  1024,
 			WriteBufferSize: 1024,
 		}
 
 		conn, err := upgrader.Upgrade(ctx.Writer, ctx.Request, nil)
 		if err != nil {
-			ctx.String(http.StatusInternalServerError, err.Error())
+			log.Printf("Failed to upgrade connection: %v", err)
 			return
```

Comment:
I'm not sure what is the return code in this case.

Reply:
The upgrader itself handles the error response code, I've added a comment about that